packages/toolkit/src/view/catalog/components/tabs/GetCatalogTab.tsx

```diff
@@ -0,0 +1,205 @@
+"use client";
+
+import * as React from "react";
+import { Nullable } from "instill-sdk";
+
+import {
+  Button,
+  DropdownMenu,
+  ScrollArea,
+  Separator,
+} from "@instill-ai/design-system";
+
+import { CodeBlock, ModelSectionHeader } from "../../../../components";
+import { defaultCodeSnippetStyles } from "../../../../constant";
+import { InstillStore, useInstillStore, useShallow } from "../../../../lib";
+import { useListCatalogFiles } from "../../../../lib/react-query-service/catalog";
+import {
+  Catalog,
+  File,
+} from "../../../../lib/react-query-service/catalog/types";
+import { env } from "../../../../server";
+import {
+  GetCatalogTabInputSchema,
+  GetCatalogTabOutputSchema,
+} from "../lib/constant";
+import { truncateName } from "../lib/helpers";
+
+type GetCatalogTabProps = {
+  catalog: Catalog;
+  isProcessed: boolean;
+  onGoToUpload: () => void;
+  namespaceId: Nullable<string>;
+};
```

Comment:
@thewbuk If this is not used in any other place, let shrink it into the named parameter